spec/beaker/dsl/install_utils/foss_utils_spec.rb

```diff
@@ -414,7 +414,7 @@ def logger
     context 'on el-5' do
       let(:platform) { Beaker::Platform.new('el-5-i386') }
       it 'installs' do
-        expect(subject).to receive(:on).with(hosts[0], /puppetlabs-release-el-5\.noarch\.rpm/)
+        expect(hosts[0]).to receive(:install_package_with_rpm).with('http://yum.puppetlabs.com/puppetlabs-release-el-5.noarch.rpm', '--replacepkgs', {:package_proxy=>false})
```

Comment:
`install_package_with_rpm` will be used by more people than just that one usage, especially as we go through refactoring, getting other parts of the system to use this method to eliminate much of our rpm usage duplication.

I think there is an implication in an `opts` hash parameter that the options are _optional_, so if this is not, I would suggest either making it a real argument of the `install_package_with_rpm` method, or making sure that you strongly point out that it will be expected in usage every time in the doc.